misc/cfg/gfx_gl_eyecandy.cfg

```diff
@@ -116,15 +121,14 @@ gl_miptexLevel               "0"
 gl_picmip                    "0"
 gl_playermip                 "0"
 gl_scaleModelTextures        "0"
-gl_scaleTurbTextures         "0"
+gl_scaleTurbTextures         "1"
 gl_subdivide_size            "64"
-gl_textureless                 "0"
-gl_texturemode               "gl_linear_mipmap_linear"
+gl_textureless               "0"
+gl_texturemode               "GL_LINEAR_MIPMAP_NEAREST"
```

Comment:
gl_linear_mipmap_linear

Reply:
No worries. I was getting the defaults from the outdated web manual, I'll lower case these.